src/world/entity/mod.rs

```diff
@@ -152,8 +257,19 @@ impl Common {
             _ => EntityType::Npc,
         };
 
+
+        let sex_sprite_path = match self.sex == Sex::Female {
+            true => "¿©",
+            false => "³²",
+        };
+
         let file_path = match entity_type {
-            EntityType::Player => format!("¸ó½ºÅÍ\\b_{}", script_loader.get_job_name_from_id(self.job_id)),
+            EntityType::Player => format!(
+                "ÀÎ°£Á·\\¸öÅë\\{}\\{}_{}",
+                sex_sprite_path,
+                get_sprite_path_for_player_job(&script_loader.get_job_name_from_id(self.job_id)),
+                sex_sprite_path
+            ),
```

Comment:
I know that this is 100% my fault but since you are adding more code to the functions could you move all the duplicate code of `new` and `reload_sprite` to a common function?